common-healing.lic

```diff
@@ -182,7 +202,19 @@ module DRCH
         wounds[severity_level] << part
       end
     end
-    wounds
+
+    parasites = Hash.new { |h, k| h[k] = [] }
+    parasites_line.split(',').each do |parasite|
+      parasite =~ all_parasites_re
+      parasite_name = Regexp.last_match.to_s
+      parasite =~ /on your (?<part>(?:left|right)?\s?(?:\w+))/
+      location = Regexp.last_match[:part].to_s
+      parasites[parasite_name] << location
+    end
+
+    health = { 'wounds' => wounds, 'poisoned' => poisoned, 'diseased' => diseased,
```

Comment:
I _think_ you could get away with 'returning' the hash (and not need the assignment to `health`), but I'm not sure if that makes it clearer or not...

Reply:
That makes sense - `health` isnt being used